MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -321,9 +323,9 @@ private async Task FetchDataFromTags(Audio audio, Model.MediaInfo.MediaInfo medi
                 }
 
                 // Save extracted lyrics if they exist,
-                // and if we are replacing all metadata or the audio doesn't yet have lyrics.
+                // and if the audio doesn't yet have lyrics.
                 if (!string.IsNullOrWhiteSpace(tags.Lyrics)
```

Comment:
Instead of passing all streams you could also just check if any is an LRC stream and only pass a bool